src/Microsoft.DotNet.Interactive.HttpRequestParser/HttpVariableDeclarationAndAssignmentNode.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#nullable enable
+
+using Microsoft.CodeAnalysis.Text;
+using System;
+
+namespace Microsoft.DotNet.Interactive.HttpRequest;
+
+internal class HttpVariableDeclarationAndAssignmentNode : HttpSyntaxNode
+{
+    public HttpVariableDeclarationNode? DeclarationNode { get; private set; }
+    public HttpVariableAssignmentNode? AssignmentNode { get; private set; }
+    public HttpVariableValueNode? ExpressionNode { get; private set; } 
```

Comment:
Calling this property `ValueNode` is probably a better match to the property's type name at this point.